go/vt/vtgate/planbuilder/operators/horizon_expanding.go

```diff
@@ -167,6 +170,24 @@ func expandOrderBy(ctx *plancontext.PlanningContext, op Operator, qp *QueryProje
 	}
 }
 
+// exposeOrderingColumn will expose the ordering column to the outer query
+func exposeOrderingColumn(ctx *plancontext.PlanningContext, qp *QueryProjection, orderBy OrderBy, derived string) OrderBy {
```

Comment:
What if the ordering is an expression that uses multiple columns, or is that explicitly unsupported?

Reply:
The expression on the inside can be made up of however complex expression, but on the outside it's still a single column. Example:

```sql
select *
from (
	select a+b+c from tbl order by a+b+c
) as t
```

Does that answer your question?

Reply:
yeh, thanks. 👍